main/typings.d.ts

```diff
@@ -11,12 +11,52 @@ export interface Activity {
 }
 
 export type FILTransaction = {
-  hash: string
-  timestamp: number
-  status: TransactionStatus
-  outgoing: boolean
-  amount: string
-  address: string
+  hash: string;
+  height: number;
+  timestamp: number;
+  status: TransactionStatus;
+  outgoing: boolean;
+  amount: string;
+  address: string;
+}
```

Comment:
This is a tricky one: The imported types don't use semicolons, `main/typings.d.ts` uses semicolons, `renderer/typings.d.ts` doesn't use semicolons. I've updated the style to be at least consistent per file. Since our linter didn't complain, I think this discussion is out of scope for this PR, I'm going to open a new issue 👍

Reply:
https://github.com/filecoin-station/filecoin-station/issues/444